.env-files/.env.test

```
@@ -4,8 +4,8 @@ SENTRY_DSN=https://0@o0.ingest.sentry.io/0
 ENVIRONMENT=development
 AUDIT_URL=https://smaug.carrot.eco
 AI_ATTACHMENT_VALIDATOR_API_URI=https://smaug.carrot.eco/attachments/validate
-DOCUMENT_ATTACHMENT_BUCKET_NAME=smaug-development-document-attachments
-DOCUMENT_BUCKET_NAME=smaug-development-methodology-executions
+DOCUMENT_ATTACHMENT_BUCKET_NAME=smaug-production-document-attachments
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DOCUMENT_ATTACHMENT_BUCKET_NAME key should go before the ENVIRONMENT key

(UnorderedKey)